package.json

```diff
@@ -61,9 +62,13 @@
     "@hmcts/ccpay-web-component": "4.0.0-beta87",
     "@hmcts/media-viewer": "1.3.1",
     "@hmcts/nodejs-healthcheck": "1.7.0",
+    "@hmcts/nodejs-logging": "^3.0.0",
+    "@hmcts/properties-volume": "0.0.9",
     "@nguniversal/express-engine": "^6.0.0",
     "@nguniversal/module-map-ngfactory-loader": "^6.0.0",
     "@nicky-lenaers/ngx-scroll-to": "^1.0.0",
+    "applicationinsights": "^1.0.8",
```

Comment:
Could this be better a higher version?

Reply:
This upgrade will impact many other dependencies. So, we can consider upgrading this along with ccd-admin-web or ccd-gateway.